### PR TITLE
Continuous delivery

### PR DESCRIPTION
**Pull request by AlexMoroz**

### CODE REVIEW

File: mockup/mockup.html [hunk 1]
```diff
@@ -27,14 +40,6 @@
 	<script src="js/timeline.js"></script>
 	<script src="js/episodes.js"></script>
 	
-	<!-- Labels-->
-	<script src="lib/label.js"></script>
```
Comment (bandle) [inline mockup/mockup.html:31]: - [ ] You have in mind that we changed sth in label.js ?

File: package.json [hunk 2]
```diff
@@ -29,5 +29,11 @@
     "grunt-contrib-jshint": "^1.0.0",
     "grunt-contrib-qunit": "^1.0.1",
     "grunt-contrib-watch": "^0.6.1"
+  },
+  "dependencies": {
+    "bootstrap": "^3.3.6",
+    "jquery": "^2.2.1",
+    "leaflet": "^0.7.7",
```
Comment (bandle) [inline package.json:36]: - [ ] We use Leaflet 1.0.0-beta.2

File: package.json [hunk 3]
```diff
@@ -29,5 +29,11 @@
...
+    "leaflet-label": "^0.2.1-0"
```
Comment (bandle) [inline package.json:37]: - [ ] Like stated above - we have a slightly modded version of this plugin

**Comment by AlexMoroz** [comment]
Ok, that's actually what I wanted to ask about)

**Pull request closed without merging**